src/driver/oracle/OracleDriver.ts

```diff
@@ -252,8 +252,8 @@ export class OracleDriver implements Driver {
      * Prepares given value to a value to be persisted, based on its column type and metadata.
      */
     preparePersistentValue(value: any, columnMetadata: ColumnMetadata): any {
-        if (columnMetadata.valueTransformer)
-            value = columnMetadata.valueTransformer.transformTo(value);
+        if (columnMetadata.transformer)
+            value = columnMetadata.transformer.to(value);
```

Comment:
You need to do better test coverage. What if `to` is not defined in `transformer`. This code will fail. Same with `from`.

Reply:
@pleerock Do you think the `ColumnMetadata` constructor would be a good place to ensure that both `from` and `to` exist on the given transformer?